### PR TITLE
chore: move domain hash names to separate crate

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Refactor out the common hash domain names between `tari` and `tari-dan` repos.

Motivation and Context
---
Constants should always be defined at one place.

How Has This Been Tested?
---

What process can a PR reviewer use to test or verify this change?
---

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

File: applications/minotari_console_wallet/Cargo.toml [hunk 1]
```diff
@@ -20,7 +20,10 @@ tari_p2p = { path = "../../base_layer/p2p", features = ["auto-update"] }
 tari_script = { path = "../../infrastructure/tari_script" }
 tari_shutdown = { path = "../../infrastructure/shutdown" }
 tari_utilities = { version = "0.7" }
-minotari_wallet = { path = "../../base_layer/wallet", features = ["bundled_sqlite"] }
+minotari_wallet = { path = "../../base_layer/wallet", features = [
+  "bundled_sqlite",
+] }
+tari_hash_domains = { git = "https://github.com/tari-project/tari_hash_domains.git" }
```
Comment (sdbondi) [inline applications/minotari_console_wallet/Cargo.toml]: We should use a specific tag here.

**Review by sdbondi — DISMISSED** [review]
utACK

Checked that hash domains are equivalent - LGTM

EDIT: Though seems to be an issue with one of the paths

**Comment by Cifko** [comment]
> utACK
> 
> Checked that hash domains are equivalent - LGTM
> 
> EDIT: Though seems to be an issue with one of the paths

Yeah, `cargo.toml` vs `Cargo.toml` not a problem on windows :)

**Review by sdbondi — APPROVED** [review]

**Comment by sdbondi** [comment]
Looks like the chat FFI tests are failing in all PRs

**Comment by Cifko** [comment]
> Looks like the chat FFI tests are failing in all PRs

yeah, I tried to run it locally, it just hangs, never finishes

**Review by SWvheerden — APPROVED** [review]
test on wallet recovery, and wallet recovered fine

**Pull request merged**